core/data/src/main/java/com/dev/innerview/core/data/repository/InnerViewRepositoryImpl.kt

```diff
@@ -0,0 +1,32 @@
+package com.dev.innerview.core.data.repository
+
+import com.dev.innerview.core.data_api.InnerViewRepository
+import com.dev.innerview.core.database.datasource.InnerViewDataSource
+import com.dev.innerview.core.model.InnerView
+import com.dev.innerview.core.model.InnerViewType
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import java.time.ZonedDateTime
+import javax.inject.Inject
+
+class InnerViewRepositoryImpl @Inject constructor(
+    private val innerViewDataSource: InnerViewDataSource
+) : InnerViewRepository {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line
<!-- __reviewdog__:Cg84NzViNWY2Y2IxMWZjNDkSBmt0bGludA== -->